README.md

```diff
@@ -1,3 +1,24 @@
 # golang-training-shop
 
-![alt text](https://github.com/dimaxdqwerty/golang-training-shop/raw/main/shop.png "shop scheme")
+## Description
+
+Application for managing products with such operations as in table below:
+
+
+|             Path            | Method | Description                           | Body example                                                                                                                                                                                                                     |
+|:---------------------------:|--------|---------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| /products                   | GET    | get all products                      | [{"ID":2,"Name":"Samsung Galaxy Buds","ProductCategoryID":1,"Quantity":8,"UnitPrice":"120,00 ?"},{"ID":3,"Name":"Samsung Electronics EVO Select 256GB MicroSDXC","ProductCategoryID":1,"Quantity":26,"UnitPrice":"30,00 ?"}] |
```

Comment:
could you show json body in the following way:
```json
Here goes your json object definition
```
I took this answer from here: https://stackoverflow.com/questions/14901245/how-to-style-a-json-block-in-github-wiki